styles/styles.css

```diff
@@ -27,17 +27,25 @@ body {
 /*** TODO: side bar ***/
 a {
```

Comment:
Change `a` to `standard-page-side-bar li a` or add a class name to the "li a" elements. There will be other "a" elements so we don't want to apply this styling to all <a> elements.